lib/msf/core/post/windows/packrat.rb

```diff
@@ -14,7 +14,28 @@ module Packrat
 
         include Msf::Post::File
         include Msf::Post::Windows::UserProfiles
-		  
```

Comment:
Separate PR to rubocop this file https://github.com/rapid7/metasploit-framework/pull/15841